docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,20 @@
+| .NET Standard              | [1.0] | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5] | [1.6] | [2.0] |
+|----------------------------|-------|--------|-------|-------|-------|-------|-------|-------|
+| .NET Core                  | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0   | 1.0   | 2.0   |
+| .NET Framework             | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1 | 4.6.1 | 4.6.1 |
```

Comment:
Ok now that I read the part of the .NET 2.0 tooling. So I'm thinking we should keep the current state of the tooling in the docs and just change it in the standard repo for now with a note that this is changing. Thoughts?